components/support/ffi/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ffi-support"
 edition = "2018"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
This is a bit awkward. We need to cut a release of this, and then do another PR updating it (and that 2nd part is required because of the Cargo.lock changes). Between when the release of ffi-support is published and when the update PR is cut, I believe `--frozen`/`--locked` builds will not properly function, since the Cargo.lock will need changing.

I filed #3463 as a possible fix for the awkwardness. The steps would be largely the same, but avoid that problem, and it's more natural to have it occur across two repos.